### PR TITLE
Remove runtime checks for Cardinal

### PR DESCRIPTION
**Pull request by sestevens**
### Summary of changes

- Remove runtime checks for CardinalMobile. Now that we have an xcframework, we no longer need the work-around for SPM, so we can import Cardinal directly.

### Checklist

- ~Added a changelog entry~

- @sestevens

### CODE REVIEW

File: SampleApps/SPMTest/SPMTest.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -417,7 +417,7 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/braintree/braintree_ios.git";
 			requirement = {
-				branch = "cardinal-xcframework";
+				branch = master;
```
Comment (sestevens) [inline SampleApps/SPMTest/SPMTest.xcodeproj/project.pbxproj:420]: Setting this back to master now that the `cardinal-xcframework` branch has been merged in.

File: Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.m [hunk 2]
```diff
@@ -112,13 +112,6 @@ - (void)handleRequest:(BTPaymentFlowRequest *)request
 
         NSError *integrationError;
 
-        if (!NSClassFromString(@"CardinalSession")) {
-            [[BTLogger sharedLogger] critical:@"CardinalMobile framework is not included. For SPM integrations, you must manually include it."];
-            integrationError = [NSError errorWithDomain:BTThreeDSecureFlowErrorDomain
-                                                   code:BTThreeDSecureFlowErrorTypeConfiguration
-                                               userInfo:@{NSLocalizedDescriptionKey: @"CardinalMobile framework is not included. For SPM integrations, you must manually include it."}];
-        }
```
Comment (sestevens) [inline Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.m:120]: Merchants will get a compile-time error if Cardinal hasn't been included correctly, so this run-time error will never be sent. It should be safe to remove it.

**Review by sshropshire — APPROVED** [review]

File: Sources/BraintreeThreeDSecure/V2UICustomization/BTThreeDSecureV2UICustomization_Internal.h [hunk 3]
```diff
@@ -4,8 +4,10 @@
 #import <BraintreeThreeDSecure/BTThreeDSecureV2UICustomization.h>
 #endif
 
+#import <CardinalMobile/CardinalMobile.h>
+
 @interface BTThreeDSecureV2UICustomization ()
 
-@property (nonatomic, strong) id cardinalValue;
+@property (nonatomic, strong) UiCustomization *cardinalValue;
```
Comment (sshropshire) [inline Sources/BraintreeThreeDSecure/V2UICustomization/BTThreeDSecureV2UICustomization_Internal.h:11]: random question: We're keeping the BT customization wrapper types?

Comment (sestevens) [reply Sources/BraintreeThreeDSecure/V2UICustomization/BTThreeDSecureV2UICustomization_Internal.h:11]: Yeah, I think they're nice to have. It lets us have a little more control over the public API. Plus, removing them now would be a breaking change.

Comment (sshropshire) [reply Sources/BraintreeThreeDSecure/V2UICustomization/BTThreeDSecureV2UICustomization_Internal.h:11]: Yeah that makes sense. There are a few areas in the Android SDK where wrapper types would make sense to keep 3rd-party deps from introducing breaking changes.

File: SWIFT_PACKAGE_MANAGER.md [hunk 4]
```diff
@@ -27,9 +27,9 @@ There is a known Xcode bug, reported in [this GitHub issue](https://github.com/b
 
 To use the `BraintreeDataCollector` library, you must also check the box for `KountDataCollector`.
 
-To use the `PayPalDataCollector`, `BraintreePaymentFlow`, `BraintreeThreeDSecure`, `BraintreePayPal`, or `BraintreeVenmo` libraries, you must also check the box for `PPRiskMagnes`.
+To use the `PayPalDataCollector`, `BraintreePaymentFlow`, `BraintreePayPal`, or `BraintreeVenmo` libraries, you must also check the box for `PPRiskMagnes`.
 
-To use the `BraintreeThreeDSecure` library, you must also check the box for `CardinalMobile`. _(This only applies to versions greater than 5.2.0. See [instructions below](#braintreethreedsecure) if you are using versions 5.0.0 to 5.2.0.)_
+To use the `BraintreeThreeDSecure` library, you must also check the boxes for `CardinalMobile` and `PPRiskMagnes`. _(This only applies to versions greater than 5.2.0. See [instructions below](#braintreethreedsecure) if you are using versions 5.0.0 to 5.2.0.)_
```
Comment (sestevens) [inline SWIFT_PACKAGE_MANAGER.md:32]: I missed that PPRiskMagnes is also required for 3DS when I opened the previous PR. Fixing that here.

**Review by sarahkoop — APPROVED** [review]

File: SWIFT_PACKAGE_MANAGER.md [hunk 5]
```diff
@@ -27,9 +27,9 @@ There is a known Xcode bug, reported in [this GitHub issue](https://github.com/b
 
 To use the `BraintreeDataCollector` library, you must also check the box for `KountDataCollector`.
 
-To use the `PayPalDataCollector`, `BraintreePaymentFlow`, `BraintreeThreeDSecure`, `BraintreePayPal`, or `BraintreeVenmo` libraries, you must also check the box for `PPRiskMagnes`.
+To use the `PayPalDataCollector`, `BraintreePaymentFlow`, `BraintreePayPal`, or `BraintreeVenmo` libraries, you must also check the box for `PPRiskMagnes`.
```
Comment (scannillo) [inline SWIFT_PACKAGE_MANAGER.md:30]: I think this list should still call out `BraintreeThreeDSecure`

File: SWIFT_PACKAGE_MANAGER.md (hunk 4 above)
Comment (scannillo) [reply SWIFT_PACKAGE_MANAGER.md:32]: If we revert the change on the line above, we can get rid of this change

Comment (sestevens) [reply SWIFT_PACKAGE_MANAGER.md:32]: Yeah, I'm just worried that someone would read "To use the `BraintreeThreeDSecure` library, you must also check the boxes for `CardinalMobile`" as meaning that you **only** need to include `CardinalMobile` and miss the fact that you also need `PPRiskMagnes`.

Comment (scannillo) [reply SWIFT_PACKAGE_MANAGER.md:32]: Yeh - that is fair. But we can keep `BraintreeThreeDSecure` in the list above still.

**Review by scannillo — APPROVED** [review]

File: SWIFT_PACKAGE_MANAGER.md (hunk 4 above)
Comment (sestevens) [reply SWIFT_PACKAGE_MANAGER.md:32]: Sure, that sounds good 👍

File: SWIFT_PACKAGE_MANAGER.md (hunk 5 above)
Comment (sestevens) [inline SWIFT_PACKAGE_MANAGER.md:30]:
```suggestion
To use the `PayPalDataCollector`, `BraintreePaymentFlow`, `BraintreeThreeDSecure`, `BraintreePayPal`, or `BraintreeVenmo` libraries, you must also check the box for `PPRiskMagnes`.
```

**Pull request merged**